service/service.py

```diff
@@ -88,7 +88,13 @@ def internal_server_error(error):
 
 @app.route('/suppliers/<int:supplierID>', methods = ['GET'])
 def read(supplierID):
```

Comment:
Would be better to name it 'get_a_supplier()'

Reply:
Thank you both have been fixed